.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
+  schedule:
+    # Daily at 3:21
+    - cron: '21 3 * * *'
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+        python-version:
+          - name: pypy3
+            toxenv: pypy3-build
+          - name: pypy3
+            toxenv: pypy3-safety
+          - name: pypy3
+            toxenv: pypy3-tests
+          - name: 3.7
+            toxenv: py37-build
+          - name: 3.7
+            toxenv: py37-safety
+          - name: 3.7
+            toxenv: py37-tests
+          - name: 3.8
+            toxenv: py38-build
+          - name: 3.8
+            toxenv: py38-safety
+          - name: 3.8
+            toxenv: py38-tests
+          - name: 3.9
+            toxenv: py39-build
+          - name: 3.9
+            toxenv: py39-safety
+          - name: 3.9
+            toxenv: py39-tests
+          - name: 3.9
+            toxenv: readme
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version.name }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version.name }}
+    - name: Ensure we have new enough versions to respect python_version
+      run: python -m pip install -U pip setuptools
+    - name: Install dependencies
+      run: brew install gmp coreutils
+      if: runner.os == 'macOS'
+    - name: Install elan
+      run: |
+          curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover-community/lean:3.23.0 -y
```

Comment:
```suggestion
          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover-community/lean:3.23.0 -y
```

Reply:
Oh, I just noticed: are you sure you want to hardcore a specific Lean version here?

Reply:
Almost certainly not, probably we want a matrix build with whatever lean versions we claim to support, but I think I copied this from elsewhere, probably mathlib-tools, which doesn't yet do that, it hardcodes one version.

(Which is probably a poor reason, I can just start doing it here I guess, so yeah will give that a shot).

I do though owe Jason a review on the commands API PR which I've procrastinated.. will probably try to do that before twiddling more here.

Thanks for the review!